### DISCUSSION TITLE
Can't able to track the source audio encoding codec in Embedded Device

### DISCUSSION THREAD

**Discussion by kpnkarthik** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1734] [Q&A]
In the Mobile Android and iOS, recording the audio and sending the streaming to the Cloud. The Audio able to retrieve on WebViewer, Android and iOS viewer. Whereas in the Embedded the audio is not able to fetch or read the data from mobile.

In Embedded device, opus codec uses the decode the audio and read the data. Some packets are coming in from the RTP receiver handler but those are not getting decoded and getting played, no audio is getting played on Sample rate used is 16khz and codec is opus. Tried setting to 48khz as well. 

Need help on this.

**Comment by niyatim23** [comment]
Hi @kpnkarthik, what are you using as a master (version, SDK)? Can you share verbose logs demonstrating the issue by running our sample application? Does the SDK have a stable network?

**Reply by kpnkarthik** [reply]
Currently we are using the embedded device for master and for viewer we are using the android and ios. I attached the SDP offer and answer.  And we need that, what are the parameters passing from mobile android ios to web viewer and what are the web viewer parameters decoded.
webrtc version used on Android: org.webrtc:google-webrtc:1.0.32006
kinesis version used on Android: com.amazonaws:aws-android-sdk-kinesisvideo:2.65.0

[SDP offer from Mobile.txt](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/files/11772285/SDP.offer.from.Mobile.txt)
[SDPanswer received from Device.txt](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/files/11772287/SDPanswer.received.from.Device.txt)

**Reply by kpnkarthik** [reply]
We have attached the log 
[kvs_runtime_master.log](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/files/11778758/kvs_runtime_master.log)
of the master device,
